dev/PushNotifications/GetNotificationEventArgs.h

```diff
@@ -26,6 +26,26 @@ namespace winrt::Microsoft::Windows::PushNotifications
                 return winrt::make<winrt::Microsoft::Windows::PushNotifications::implementation::PushNotificationReceivedEventArgs>(payloadAsWstring);
             }
         }
-        return GetArgsFromComStore();
+
+        const DWORD receiveArgsTimeoutInMSec{ 2000 };
+        THROW_HR_IF(HRESULT_FROM_WIN32(ERROR_TIMEOUT), !GetWaitHandleForArgs().wait(receiveArgsTimeoutInMSec));
+
+        // If COM static store was uninit, let it throw
+        if (PushNotificationHelpers::IsPackagedAppScenario())
+        {
+            return winrt::Windows::ApplicationModel::Core::CoreApplication::Properties().Lookup(ACTIVATED_EVENT_ARGS_KEY);
+        }
+        else
+        {
+            return winrt::Windows::ApplicationModel::Core::CoreApplication::Properties().Lookup(LRP_ACTIVATED_EVENT_ARGS_KEY);
```

Comment:
This key is true for the COM activation case and ShellExecute case from LRP right?

Reply:
Will add a comment pointing to open bug. https://github.com/microsoft/WindowsAppSDK/issues/2075